src/software/thunderscope/thread_safe_buffer.py

```diff
@@ -0,0 +1,90 @@
+import queue
+from software.logger.logger import createLogger
+
+
+class ThreadSafeBuffer(object):
```

Comment:
is this better than pybinding?

Reply:
i don't know tbh, python queue is pretty solid
but it is an interesting thought